### ISSUE TITLE
Add the flatpak build to flathub

### ISSUE THREAD

**Issue by Llama037** [fn2006/PollyMC#120]
### Role

I use flatpaks and play Minecraft

### Suggestion

I want PollyMC to be added to flathub for ease of downloading 

### Benefit

It would be easier to download the launcher for people that dont want to use the command line

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Launchers like Legacy launcher are on Flathub, so i don't see problems with having PollyMC on Flathub.

Also PollyMC could benefit from this, since people who don't know this launcher will be able to find it while browsing Flathub.

**Comment by iveyxen** [comment]
TL;DR nothing new, just expanding original post

That might involve a potential recursive lawsuit on every entity that supply an ability to get a launcher(flathub, fedora, other distros that ship flathub, etc.). Remind you that fedora codecs situation, where they removed some codecs without a precedent i.e. because of a _potential_ problems with law on a horizon.

But there's already a lot of other "unsanctioned distribution" launchers on flathub and no problems so far.

That change would bring the ease and accessibility, so I agree.

**Comment by Llama037** [comment]
I've genuinelly didn't know that Tl Legacy on flathub requires licence, and i thought that it was allowed as it is, but now i see that submitting polly to flathub probably isn't a viable option.

**Comment by DraconicNEO** [comment]
> I've genuinelly didn't know that Tl Legacy on flathub requires licence, and i thought that it was allowed as it is, but now i see that submitting polly to flathub probably isn't a viable option.

Why not just have it in its own flatpak repo that isn't flathub, then people can still add it to the store and download updates for it there.
